elevenclock/lang/lang_ko.py

```diff
@@ -22,40 +22,40 @@
 }
 
 lang_3 = lang_3_1 | {
-    "Hide the clock during 10 seconds when clicked": "클릭 시 10초간 시계 숨기기",
-    "Enable low-cpu mode": "저성능 모드 사용",
-    "You might lose functionalities, like the notification counter or the dynamic background": "알림 카운터, 동적 배경 등의 기능이 비활성화됩니다",
+    "Hide the clock during 10 seconds when clicked": "클릭하면 10초 동안 시계 숨기기",
+    "Enable low-cpu mode": "저성능 CPU 모드 사용",
+    "You might lose functionalities, like the notification counter or the dynamic background": "알림 카운터 또는 동적 배경과 같은 기능을 잃을 수 있습니다.",
     "Clock position and size:": "시계 위치 및 크기:",
-    "Clock size preferences, position offset, clock at the left, etc.": "시계 크기, 위치 조정, 왼쪽에 표시하기 등",
-    "Reset monitor blacklisting status": "모니터 블랙리스트 초기화",
+    "Clock size preferences, position offset, clock at the left, etc.": "시계 크기 환경 설정, 위치 조정, 왼쪽 시계 등",
+    "Reset monitor blacklisting status": "모니터 블랙리스트 상태 초기화",
     "Reset": "초기화",
-    "Third party licenses": "제3자 라이선스",
+    "Third party licenses": "타사 라이센스",
     "View": "보기",
     "ElevenClock": "ElevenClock",
     "Monitor tools": "모니터 도구",
-    "Blacklist this monitor": "이 모니터를 블랙리스트에 추가",
-    "Third Party Open-Source Software in Elevenclock {0} (And their licenses)": "ElevenClock {0}에 사용된 제3자 오픈 소스 소프트웨어 (및 그 라이선스)",
-    "ElevenClock is an Open-Source application made with the help of other libraries made by the community:": "ElevenClock은 오픈 소스 애플리케이션이며 아래 커뮤니티 라이브러리들을 활용하여 제작되었습니다:",
+    "Blacklist this monitor": "이 모니터 블랙리스트 지정",
+    "Third Party Open-Source Software in Elevenclock {0} (And their licenses)": "ElevenClock {0}의 타사 오픈 소스 소프트웨어 (및 해당 라이센스)",
+    "ElevenClock is an Open-Source application made with the help of other libraries made by the community:": "ElevenClock은 커뮤니티에서 만든 다른 라이브러리의 도움을 받아 만든 오픈 소스 응용 프로그램입니다",
     "Ok": "확인",
-    "More Info": "자세히",
+    "More Info": "추가 정보",
     "About Qt": "Qt 정보",
     "Success": "성공",
-    "The monitors were unblacklisted successfully.": "모든 모니터를 블랙리스트에서 제거하였습니다.",
-    "Now you should see the clock everywhere": "이제 모든 모니터에 시계가 표시될 것입니다",
+    "The monitors were unblacklisted successfully.": "모니터가 블랙리스트 해제되었습니다.",
+    "Now you should see the clock everywhere": "이제 당신은 어디에서나 시계를 볼 수 있습니다.",
     "Ok": "확인",
-    "Blacklist Monitor": "모니터 블랙리스트에 추가",
-    "Blacklisting a monitor will hide the clock on this monitor permanently.": "블랙리스트에 추가된 모니터에는 시계가 표시되지 않습니다.",
-    "This action can be reverted from the settings window. under <b>Clock position and size</b>": "이 선택은 설정 창의 <b>시계 위치 및 크기</b>에서 되돌릴 수 있습니다.",
-    "Are you sure do you want to blacklist the monitor \"{0}\"?": "모니터 \"{0}\"을(를) 블랙리스트에 추가하시겠습니까?",
+    "Blacklist Monitor": "블랙리스트 모니터",
```

Comment:
다이얼로그 제목으로서 단독 사용되는 텍스트인 만큼, 단어 대 단어의 건조한 번역보다는 기능을 설명하는 완전한 문장으로 표기하는 것이 자연스럽다고 생각합니다.

This text is used as a title of the dialog, so I think it should be translated as one sentence that describes the function of it instead of the dry word-by-word translation.